generator/ServiceClientGeneratorLib/Generators/Marshallers/CborResponseUnmarshaller.tt

```diff
@@ -162,51 +162,38 @@ namespace <#=this.Config.Namespace #>.Model.Internal.MarshallTransformations
 
 #>
             using (var contextCopy = new CborUnmarshallerContext(streamCopy, true, context.ResponseData))
+            {
+<#
+                GenerateAWSQueryCompatibleBlock();
```

Comment:
Is it intentional to always generate the query compatible block for CBOR services? Before we were checking for `this.Config.ServiceModel.IsAwsQueryCompatible` first.

Reply:
So I read the description after... But now a different question, if a service migrates from JSON to CBOR wouldn't the query compatible block be unnecessary?